pxtblocks/fields/field_gridpicker.ts

```diff
@@ -86,6 +88,11 @@ namespace pxtblockly {
             let tableContainerDom = tableContainer.getElement();
             if (tableContainerDom) {
                 let menuItemsDom = tableContainerDom.childNodes;
+                try {
+                    let firstItem = menuItemsDom[0].childNodes[0] as HTMLElement
+                    firstItem.className += " goog-menuitem-highlight"
+                } catch (e) {
```

Comment:
What is this trying to catch?

Reply:
Case where menuItemsDom is an empty array.  E.g., when you enter text in the search box such that there are no search results (like "foo").

Reply:
Could do `if (menuItemsDom.length > 0 && menuItemsDom[0].childNodes.length > 0)` or something but that seems more cumbersome.

Reply:
I think it's worth to use the `if`, to avoid accidentally swallowing exceptions (e.g. if in the future we add code to the `try`)

You can use `if (menuItemsDom.length && menuItemsDom[0].childNodes)` for slightly more compact code